### ISSUE TITLE
App crash on startup ( 'Primary key property 'code' does not exist on object 'Nutriment'')

### ISSUE THREAD

**Issue by aleene** [openfoodfacts/openfoodfacts-ios#345]
**Summary:** 
Xcode console:
2019-10-04 19:49:49.743955+0200 OpenFoodFacts[907:134539] *** Terminating app due to uncaught exception 'RLMException', reason: 'Primary key property 'code' does not exist on object 'Nutriment''

**Steps to reproduce:** 
Implement first solution to #343. Then start on device with xcode 11

**Observed behavior:** 
Crash

**Device and iOS version:** 
xcode and ipad

**Comment by aleene** [comment]
It is an issue in Realm. Does this library need an update to IOS 13?

**Comment by aleene** [comment]
Related to this? https://github.com/realm/realm-cocoa/issues/6264

**Comment by SuzGupta** [comment]
I was going to say it's related to this: https://github.com/realm/realm-cocoa/issues/6163. There are some suggestions in the comment thread.

**Comment by SuzGupta** [comment]
I tried changing the Realm line in `Cartfile` to `github "realm/realm-cocoa" >= 3.18.0` and then rerunning `carthage bootstrap --platform iOS --cache-builds`. This took quite a while since pretty much everything was "falling back to building from source" because the compiler in Xcode 11 is newer than what the binaries were built with. I then opened the project in Xcode and cleaned the build folder, but alas, I still see the error once the app loads into the Simulator.

**Comment by SuzGupta** [comment]
Ah, I realized I was not using the correct Carthage command. Looking at https://www.chrisjmendez.com/2016/10/30/carthage-cheat-sheet/, I tried `carthage update --platform ios --no-use-binaries --verbose`. This plus a couple Option + Product>Clean Build Folder passes did resolve this issue for me, although I now have a different crash in the Simulator. I'll open a pull request for this little change to the `Cartfile` and discuss the other issue in Slack.
